src/com/carolinarollergirls/scoreboard/utils/Version.java

```diff
@@ -19,14 +22,19 @@ public static boolean load() throws IOException {
             throw ioE;
         }
         try { releaseIs.close(); } catch ( Exception e ) { }
-        versionRelease = versionProperties.getProperty(VERSION_RELEASE_KEY);
+        Map<String, String> m = new HashMap<>();
+        for (String k : versionProperties.stringPropertyNames()) {
+          m.put(k, versionProperties.getProperty(k));
+        }
+        props = Collections.unmodifiableMap(m);
         Logger.printMessage("Carolina Rollergirls Scoreboard version "+Version.get());
         return true;
     }
 
-    public static String get() { return versionRelease; }
+    public static String get() { return props.getOrDefault(VERSION_RELEASE_KEY, "unset"); }
```

Comment:
`getOrDefault` is not available in Java 7.

Reply:
Fixed.